### PR TITLE
GitHub actions enhancement

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold 

## Related Issues
fixes: https://github.com/demisto/etc/issues/25708

## Description
Added `Github-get-github-actions-usage` command to Github integration.

### CODE REVIEW

**Comment by IkaDemisto** [comment]
[GetGithubActionsUsage.yml.zip](https://github.com/demisto/content/files/4973707/GetGithubActionsUsage.yml.zip)
Attached script for xsoar Github workflows usage widget.

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 1]
```diff
@@ -2586,11 +2588,27 @@ script:
...
-  subtype: python3
+  - arguments:
+    - default: false
+      description: Repository owner.
+      isArray: false
+      name: owner
+      required: true
+      secret: false
+    deprecated: false
+    description: Get github actions workflows usage of private repositories.
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:2599]: is this specific to private repositories? what about the public ones?

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 2]
```diff
@@ -1137,6 +1137,92 @@ def get_download_count():
...
+            workflow_name = workflow.get('name', '')
+            workflow_usage = get_workflow_usage(owner_name=owner_name, repository_name=repository_name,
+                                                workflow_id=workflow_id)
+
+            if workflow_usage:
+                usage_result.append({
+                    'workflow_name': workflow_name,
+                    'workflow_id': workflow_id,
+                    'repository': repository_name,
+                    'workflow_usage': workflow_usage
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1214]: add all 4 to command output

Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1214]:
```suggestion
                    'workflow_name': workflow_name,
                    'workflow_id': workflow_id,
                    'repository': repository_name,
                    'workflow_usage': workflow_usage,
```

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 3]
```diff
@@ -1137,6 +1137,92 @@ def get_download_count():
...
+                })
+
+    ec = {
+        'GitHub.ActionsUsage': usage_result
+    }
+    human_readable = tableToMarkdown('Github Actions Usage', usage_result,
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1220]: why not use `string_to_table_header` as transformer?

**Comment by yaakovi** [comment]
> [GetGithubActionsUsage.yml.zip](https://github.com/demisto/content/files/4973707/GetGithubActionsUsage.yml.zip)
> Attached script for xsoar Github workflows usage widget.

why not include these in the PR?

File: Packs/GitHub/Integrations/GitHub/GitHub.yml (hunk 1 above)
Comment (IkaDemisto) [reply Packs/GitHub/Integrations/GitHub/GitHub.yml:2599]: Yes, specific for private repos. Public repos Github actions are free and ```/repos/:owner/:repo/actions/workflows/:workflow_id/timing``` endpoint is not returning any data for public repos

**Comment by IkaDemisto** [comment]
> > [GetGithubActionsUsage.yml.zip](https://github.com/demisto/content/files/4973707/GetGithubActionsUsage.yml.zip)
> > Attached script for xsoar Github workflows usage widget.
> 
> why not include these in the PR?

I didn't find a way to pass arguments to script that creates the widget, `demisto` is hard coded there as repos org owner.

**Comment by yaakovi** [comment]
@kirbles19 
a new command, please review lines 2591-2616 in the yml (and release notes)

**Comment by kirbles19** [comment]
@IkaDemisto Done.

**Pull request merged**